architecture/live-queries.md

```diff
@@ -141,13 +141,3 @@ Testing scalability & reliability for live-queries with websockets has been a ch
 
 ## Benefits of this approach
 Hasura makes live-queries easy and accessible. The notion of queries is easily extended to live-queries without any extra effort on the part of the developer using GraphQL queries. This is the most important thing for us.
-
-1. Expressive/featureful live queries with full support for Postgres operators/aggregations/views/functions etc
```

Comment:
I didn't mean to delete this section. Sorry for the misunderstanding. You added a full stop to each of the points, and I meant that these full stops are not necessary. So basically this part should keep unchanged 🙂

Reply:
Sorry i will add it again.